packages/core/src/behaviors.ts

```diff
@@ -183,8 +178,6 @@ export function spawnBehavior<TEvent extends EventObject, TEmitted>(
     _event: null as any
   };
 
-  state = behavior.start ? behavior.start(actorCtx) : state;
```

Comment:
Same argument as [here](https://github.com/statelyai/xstate/pull/2916/files#r777570632). We can't execute `start` when rendering.